content/guides/dogstatsd.md

```diff
@@ -25,315 +25,331 @@ sidebar:
 ---
 
 <p class="aside">
-  This tutorial will walk you through instrumenting your application to send
-  custom metrics to Datadog. If you need some help as you go, pop by
-  <a href="irc://irc.freenode.net/datadog">#datadog on freenode</a>,
-  where we'll be happy to answer any questions you might have. (There's a
-  <a href="http://webchat.freenode.net/?randomnick=1&channels=datadog&prompt=1">
-  web chat client, too</a>.)
+This page explains what DogStatsD is, how it works, and what data it accepts.
 </p>
 
-The easiest way to get your custom metrics into Datadog is to send them to DogStatsD,
-a metrics aggregation server bundled with the Datadog Agent (in versions 3.0
-and above).
-DogStatsD implements the
-<a href="https://github.com/etsy/statsd">StatsD</a>
-protocol, along with a few extensions for special Datadog features.
+The easiest way to get your custom application metrics into Datadog is to send them to DogStatsD, a metrics aggregation daemon bundled with the Datadog Agent. DogStatsD implements 
+the <a href="https://github.com/etsy/statsd">StatsD</a> protocol and adds a few Datadog-specific
+extensions:
+
+* Histogram and Set metric types
+* Service checks and Events
+* Tagging
 
 ## How It Works
 
-DogStatsD accepts custom application metrics points over
-<a href="http://en.wikipedia.org/wiki/User_Datagram_Protocol">UDP</a>,
-and then periodically aggregates and forwards the metrics to Datadog, where
-they can be graphed on dashboards. Here's a pretty standard DogStatsd setup:
+DogStatsD accepts your custom metrics, events, and service checks over UDP and periodically aggregates and forwards them to Datadog. Because it uses UDP, your application 
+can fire metrics at DogStatsD and resume its work without waiting for a response. If DogStatsD ever crashes 
+or begins to perform poorly, your application won't skip a beat.
+
 <p>
 <img src="/static/images/dogstatsd.png"/>
 </p>
 
-### Aggregation
+As it receives data, DogStatsD aggregates multiple data points for each unique metric into a single
```

Comment:
Is this accurate it? Doesn't it emit multiple quantiles/percentiles per datapoint?